lib/dao.js

```diff
@@ -1324,7 +1325,7 @@ DataAccessObject._normalize = function(filter) {
 function DateType(arg) {
   var d = new Date(arg);
   if (isNaN(d.getTime())) {
-    throw new Error('Invalid date: ' + arg);
+    throw new Error(g.f('Invalid date: ' + arg));
```

Comment:
Missing the arg replacement
Can use g.d here for date
https://www.npmjs.com/package/strong-globalize#gformatdatevalue-options